docker/base-builder/Dockerfile

```diff
@@ -34,5 +34,10 @@ COPY --from=base-clang /usr/local/lib/clang /usr/local/lib/clang
 COPY --from=base-clang /usr/local/lib/libc++*.a /usr/local/lib/
 COPY --from=base-clang /usr/local/include/ /usr/local/include
 
-# Use libc++ thoughout the build.
-ENV CXXFLAGS -stdlib=libc++
+# These environment variables are set in the images for OSS-Fuzz benchmarks and
+# cannot be unset. Set them here as well for consistency.
```

Comment:
Consider also saying in the comment what is going to use these flags.

Reply:
Done.